### PR TITLE
Change int to int32 in API

### PR DESCRIPTION
**Pull request by justinsb**
We shouldn't be using the variable-sized int in the API

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1483)

### CODE REVIEW

**Comment by kris-nova** [comment]
Agree with defining the int size - but wondering why 32?

File: upup/pkg/kutil/import_cluster.go [hunk 1]
```diff
@@ -700,7 +700,7 @@ func (u *UserDataConfiguration) ParseBool(key string) *bool {
 	return fi.Bool(false)
 }
 
-func (u *UserDataConfiguration) ParseInt(key string) (*int, error) {
+func (u *UserDataConfiguration) ParseInt64(key string) (*int64, error) {
```
Comment (kris-nova) [inline upup/pkg/kutil/import_cluster.go]: Is anything calling this? I don't see this anywhere else in the PR..

Comment (justinsb) [inline upup/pkg/kutil/import_cluster.go]: No, I should probably just delete it...

**Comment by justinsb** [comment]
@kris-nova I chose 32 mostly because I thought 64 would be overkill for these cases.  Are there any in particular you think should be 64?

**Pull request merged**